salt/metalk8s/kubernetes/kubelet/standalone.sls

```diff
@@ -14,8 +14,13 @@ Create kubelet service environment file:
     - makedirs: True
     - dir_mode: 750
     - context:
-        options: {{ kubelet.service.options }}
-        node_ip: {{ grains['metalk8s']['control_plane_ip'] }}
+    - context:
+        options:
+      {%- for opt, value in kubelet.service.options.items() %}
+          {{ opt }}: {{ value }}
+      {%- endfor %}
+          node_ip: {{ grains['metalk8s']['control_plane_ip'] }}
```

Comment:
Is it `node_ip`? Before, it was `node-ip`...

Reply:
Oh yes agree

Reply:
Good catch!

Seems like it didn't broke the expansion (in a very visible way at least);